src/plugins/dashboard/public/application/top_nav/show_share_modal.tsx

```diff
@@ -104,21 +110,50 @@ export function ShowShareModal({
     );
   };
 
+  const rawDashboardState = stateToRawDashboardState({
+    state: currentDashboardState,
+    version: kibanaVersion,
+  });
+
+  const locatorParams: DashboardAppLocatorParams = {
+    dashboardId: savedDashboard.id,
+    filters: rawDashboardState.filters,
+    preserveSavedFilters: true,
+    query: rawDashboardState.query,
+    savedQuery: rawDashboardState.savedQuery,
+    useHash: false,
+    panels: rawDashboardState.panels,
+    timeRange,
+    viewMode: ViewMode.VIEW, // For share locators we always load the dashboard in view mode
+    refreshInterval: undefined, // We don't share refresh interval externally
+    options: rawDashboardState.options,
+  };
+
   share.toggleShareContextMenu({
     isDirty,
     anchorElement,
     allowEmbed: true,
     allowShortUrl: dashboardCapabilities.createShortUrl,
     shareableUrl: setStateToKbnUrl(
       '_a',
-      stateToRawDashboardState({ state: currentDashboardState, version: kibanaVersion }),
+      rawDashboardState,
       { useHash: false, storeInHashQuery: true },
       unhashUrl(window.location.href)
     ),
     objectId: savedDashboard.id,
     objectType: 'dashboard',
     sharingData: {
-      title: savedDashboard.title,
+      title:
+        savedDashboard.title ||
+        i18n.translate('dashboard.share.defaultDashboardTitle', {
+          defaultMessage: 'Dashboard [{date}]',
+          values: { date: moment().toISOString(true) },
```

Comment:
Does this date string need to be  converted to the user's timezone and formatted per the time format in Advanced Settings?

Reply:
Whether we issue a request for v1 or v2 reports depends on the `jobProviderOptions` that are passed in. If they contain `locatorParams` then we know it's a v2 report type. To answer your question: yes v1 can be accessed from this UI and depends on the values passed in from the consumer.

I decided to keep it this way because I was not sure whether other parts of Kibana (like Canvas and visualize) still depend on v1 report types based on the params they pass in. Once they've all been updated we should remove this funcitonality - and probably send out a message that Reports now requires locators. WDYT?